ui_advanced_config.py

```diff
@@ -243,7 +313,7 @@ def BuildUI(self):
         self.objectsDensityLabel.grid(row=9, column=1, sticky="nsew")
         self.objectsDensityEntry.grid(row=9, column=2, sticky="nsew")
         self.objectsCompIDsLabel.grid(row=10, column=1, sticky="nsew")
-        self.objectsCompIDsEntry.grid(row=10, column=2, sticky="nsew")
+        self.objectsCompIDsCombo.grid(row=10, column=2, sticky="nsew")
         self.objectsPointsCountLabel.grid(row=11, column=1, sticky="nsew")
         self.objectsPointsCountEntry.grid(row=11, column=2, sticky="nsew")
         self.objectsUpdateButton.grid(
```

Comment:
On update without changing any values, "points_count" attribute of object id 0 has changed from "false" to "0". Picture attached.

![image](https://github.com/Leading-AI-Development-Solutions/MAIA/assets/97774046/08ea57b6-337f-4b6a-9c6f-2c90f5296df4)